### PR TITLE
Add syntax highlighting support for ruby and erb

### PR DESCRIPTION
**Pull request by colebemis**
This PR adds support for `ruby` and `erb` syntax highlighting in code examples:

## Example

````
```erb
<h1 class="hi">foo</h1>
<%= render(Primer::SpinnerComponent.new) %>
```
````

### Before 

image

### After

image

Closes https://github.com/primer/doctocat/issues/200

### CODE REVIEW

**Comment by colebemis** [comment]
Looks like this is working in the Primer ViewComponents docs for one-line examples:

image

...but it has a hard time with multi-line examples:

image

**Comment by colebemis** [comment]
I filed an issue in the prism repo to report the bug described in my previous comment: https://github.com/PrismJS/prism/issues/2602

**Comment by manuelpuyol** [comment]
@colebemis any updates here?

**Comment by colebemis** [comment]
@manuelpuyol Sorry, I haven't had time to follow up on this. If you feel comfortable, feel free to take over this PR. Otherwise, I'll revisit this when I have more time.

**Comment by manuelpuyol** [comment]
Digged some more and I think I found the problem https://github.com/FormidableLabs/prism-react-renderer/issues/104

**Comment by manuelpuyol** [comment]
As an update, my PR https://github.com/FormidableLabs/prism-react-renderer/pull/105 was merged! I'm waiting for `prism-react-renderer` to publish a new version with the fix.

For it to work properly we have to set up our custom PrismJS, which I did in `theme/src/prism.js`, but that means that we need to manually setup the languages we want to support. @colebemis mind helping me here how to proceed?
I

File: theme/src/github.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+export default {
```
Comment (manuelpuyol) [inline theme/src/github.js:1]: `prism-react-rendered` changed the `github` theme on v1.2.0 and it does not look right:

| Before | After |
|--------|--------|
| image | image |

So I decided to extract the previous version of the theme into this project to keep the docs consistent

**Review by colebemis — COMMENTED** [review]
Wow, nice work, @manuelpuyol! Very impressed with your debugging abilities. Thanks for pushing this over the finish line ❤️  

Approved ✅ (I can't officially approve because I started the PR 😅)

**Pull request merged**